### PR TITLE
Support DAML SDK v0.13.55

### LINKED ISSUE #92: Support DAML SDK 0.13.55

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Closes #92

### CODE REVIEW

File: grpc-codegen.sh [hunk 1]
```diff
@@ -22,8 +22,10 @@ trap clean ERR
 if [ ! -d "$PROTO_PATH" ]; then
     mkdir -p "$PROTO_PATH" && (
       cd "$PROTO_PATH"
-      curl -s https://digitalassetsdk.bintray.com/DigitalAssetSDK/com/digitalasset/ledger-api-protos/"$SDK_VERSION"/ledger-api-protos-"$SDK_VERSION".tar.gz | tar xz --strip-components 2
-      curl -s https://digitalassetsdk.bintray.com/DigitalAssetSDK/com/digitalasset/daml-lf-archive-protos/"$SDK_VERSION"/daml-lf-archive-protos-"$SDK_VERSION".tar.gz | tar xz --strip-components 3
+      curl -s https://repo1.maven.org/maven2/com/digitalasset/ledger-api-protos/$SDK_VERSION/ledger-api-protos-$SDK_VERSION.tar.gz | tar xz --strip-components 2
+      curl -s https://digitalassetsdk.bintray.com/DigitalAssetSDK/com/digitalasset/daml-lf-1.6-archive-proto/"$SDK_VERSION"/daml-lf-1.6-archive-proto-"$SDK_VERSION".tar.gz | tar xz
```
Comment (cocreature) [inline grpc-codegen.sh:26]: Why do we use 1.6?

File: src/codec/ValueCodec.ts [hunk 2]
```diff
@@ -23,8 +23,8 @@ export const ValueCodec: Codec<PbValue, Value> = {
             return {valueType: 'contractId', contractId: value.getContractId()};
         } else if (value.hasDate()) {
             return {valueType: 'date', date: '' + value.getDate()};
-        } else if (value.hasDecimal()) {
-            return {valueType: 'decimal', decimal: value.getDecimal()};
+        } else if (value.hasNumeric()) {
```
Comment (cocreature) [inline src/codec/ValueCodec.ts:26]: Is this right? Value can be a numeric that is not a decimal.

**Comment by stefanobaghino-da** [comment]
Since a few extra clean up steps have been necessary, the reviewer is advised to review commit-by-commit to make the changes clear.

File: grpc-codegen.sh (hunk 1 above)
Comment (stefanobaghino-da) [reply grpc-codegen.sh:26]: @bame-da Do we want to move to 1.7 or ask users to work with 1.6?

File: src/codec/ValueCodec.ts (hunk 2 above)
Comment (stefanobaghino-da) [reply src/codec/ValueCodec.ts:26]: Are numeric part of 1.6 already?

Comment (cocreature) [reply src/codec/ValueCodec.ts:26]: No but they are part of 1.7 which is part of 0.13.55. If I use the bindings to listen on a transaction stream, I should be able to receive a numeric. What will happen in that case?

Comment (stefanobaghino-da) [reply src/codec/ValueCodec.ts:26]: Ok, sorry for the confusion here. This is an API Numeric, which is going to be always be a string. So this should not really be a problem. Makes sense?

Comment (cocreature) [reply src/codec/ValueCodec.ts:26]: kinda, you are still setting `valueType` to `decimal` which is wrong if you don’t have a Numeric 10. But happy to keep it for now. Maybe just add a comment.

Comment (stefanobaghino-da) [reply src/codec/ValueCodec.ts:26]: I would prefer existing code to not break regarding this. If we want, we can change it later.

File: grpc-codegen.sh (hunk 1 above)
Comment (bame-da) [reply grpc-codegen.sh:26]: If 1.7 is not fully supported, let's stick to 1.6 for now.

**Pull request merged**